internal/image/validate.go

```diff
@@ -107,9 +107,11 @@ func ValidateImage(ctx context.Context, comp app.SnapshotComponent, p policy.Pol
 	}
 
 	var allResults []evaluator.Outcome
-	for _, e := range a.Evaluators {
+
+	for _, e := range evaluators {
 		// Todo maybe: Handle each one concurrently
 		results, data, err := e.Evaluate(ctx, []string{inputPath})
+		log.Debug("\n\nRunning conftest policy check\n\n")
```

Comment:
Nit, this feels a bit noisy without adding any information useful for debugging, perhaps:

```suggestion
	for i, e := range evaluators {
		// Todo maybe: Handle each one concurrently
		log.Debugf("Running evaluator %d over %q", i, inputPath)
		results, data, err := e.Evaluate(ctx, []string{inputPath})
```